rosetta/src/main/java/com/ahmedjazzar/rosetta/LocalesUtils.java

```diff
@@ -265,7 +265,8 @@ static boolean setLocale(Locale newLocale, Activity activity) {
         }
 
         if (LocalesUtils.setAppLocale(activity.getApplicationContext(), newLocale))   {
-            LocalesUtils.refreshApplication(activity);
+            //LocalesUtils.refreshApplication(activity);
```

Comment:
Just remove it.

Reply:
when i was using your refresh method the problem was that it was recreating application and current activity both.
i just wanted to recreate whole application. so it can recreate all resources in my application

Reply:
Cool with me, but just no need to comment, removing the line is enough.